PORTAL/jobs/__init__.py

```diff
@@ -78,11 +76,12 @@ class Jobs:
 
     FS = PortalFS
 
-    def __init__(self, cfg, *, devmode=False):
+    def __init__(self, worker_name, cfg, *, devmode=False):
         self._cfg = cfg
         self._devmode = devmode
-        self._fs = self.FS(cfg.data_dir)
-        self._workers = _workers.Workers.from_config(cfg)
+        self._fs = self.FS(worker_name, cfg.data_dir)
+        self._workers = _workers.Workers.from_config(cfg.workers)
```

Comment:
FWIW, my gut says it would be better to pass the whole config rather than just the set of worker configs.